scripts/lib/libvirt/admin-config.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+
+import libvirt
+import argparse
+import libvirt_helpers
+from string import Template
+
+conn = libvirt.open("qemu:///system")
+if not conn:
+    print("Failed to open connection to the hypervisor")
+    sys.exit(1)
+
+parser = argparse.ArgumentParser(description="Create Admin Node Config.")
+parser.add_argument("cloud", type=str, help="Name of the cloud")
+parser.add_argument("adminnodememory", type=str, help="Amount of Memory")
+parser.add_argument("adminvcpus", type=str, help="Number of VCPUs")
+parser.add_argument("emulator", type=str, help="Emulator")
+parser.add_argument("adminnodedisk", type=str, help="Disk of the Admin Node")
+parser.add_argument("localrepositorymount", type=str, help="Local Repository Mount XML")
+
+args = parser.parse_args()
+
+def main():
+    cloud = args.cloud
+    cpuflags = libvirt_helpers.cpuflags()
+    fout = open("/tmp/{0}-admin.xml".format(cloud), "w")
```

Comment:
This file is not used until the end.  I would suggest to use `with` statement:

```python
with open("/tmp/{0}-admin.xml".format(cloud), "w") as f:
    f.write(xml)
```

http://effbot.org/zone/python-with-statement.htm